### DISCUSSION TITLE
return_pred_int for AutoArima

### DISCUSSION THREAD

**Discussion by deinega** [alan-turing-institute/sktime#3494] [Q&A]
Does `return_pred_int` exist in `AuroArima`, as it is written in documentation?
https://www.sktime.org/en/v0.8.0/api_reference/auto_generated/sktime.forecasting.arima.AutoARIMA.html
Trying to use it results in an error that it does not exist

**Comment by fkiraly** [comment]
current version is 0.13.4, you need to look at the most recent docs.

The `return_pred_int` argument was replaced by the method `predict_interval`.

**Comment by deinega** [comment]
Thanks you!
